doc/classes/Control.xml

```diff
@@ -500,7 +500,7 @@
 			<argument index="1" name="type" type="StringName" default="&quot;&quot;">
 			</argument>
 			<description>
-				Returns a [StyleBox] from assigned [Theme] with given [code]name[/code] and associated with [Control] of given [code]type[/code].
+				Returns a [StyleBox] from assigned [Theme] with the given [code]name[/code] on the current [Control] of the given [code]type[/code]. e.g. calling [code]get_theme_stylebox("panel")[/code] will return the stylebox named "panel" on the current control.
```

Comment:
```suggestion
				Returns a [StyleBox] from assigned [Theme] with the given [code]name[/code] on the current [Control] of the given [code]type[/code]. E.g. calling [code]get_theme_stylebox("panel", "PopupDialog")[/code] will return the [code]panel[/code] [StyleBox] defined a theme property in [PopupDialog].
```

There's a number of other `get_theme_*` methods with similar phrasing and API which should be similarly improved IMO.